packages/indicators/src/__tests__/__integration/arithmeticAnalytics.fixtures.json

```diff
@@ -0,0 +1,447 @@
+      {
+        "surveyCode": "Births",
+        "entityCode": "GR",
+        "submission_time": "2020-11-30T15:00:00Z",
+        "answers": {
+          "Female": "3",
+          "Male": "4"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "GR",
+        "submission_time": "2020-12-29T15:00:00Z",
+        "answers": {
+          "Female": "60",
+          "Male": "80"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "GR",
+        "submission_time": "2020-12-30T15:00:00Z",
+        "answers": {
+          "Female": "6",
+          "Male": "8"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "IT",
+        "submission_time": "2019-06-05T15:00:00Z",
+        "answers": {
+          "Female": "10",
+          "Male": "20"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "IT",
+        "submission_time": "2020-07-07T15:00:00Z",
+        "answers": {
+          "Female": "30",
+          "Male": "40"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "ES",
+        "submission_time": "2020-01-01T15:00:00Z",
+        "answers": {
+          "Female": "100"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "ES",
+        "submission_time": "2020-02-02T15:00:00Z",
+        "answers": {
+          "Female": "150",
+          "Male": "200"
+        }
+      }
+    ]
+  },
+  "testCases": [
+    {
+      "description": "Throws if config is empty",
+      "input": {
+        "indicatorCodes": ["EmptyConfig"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2020-12-31",
+          "organisationUnitCodes": ["TO"]
+        }
+      },
+      "throws": true,
+      "expected": "Should not be empty"
+    },
+    {
+      "description": "Returns empty results if org unit has no data",
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2020-12-31",
+          "organisationUnitCodes": ["TO"]
+        }
+      },
+      "expected": []
+    },
+    {
+      "description": "Returns empty results if date range has no data",
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2017-01-01",
+          "endDate": "2017-12-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": []
+    },
+    {
+      "description": "Evaluates an arithmetic formula",
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2019-01-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": [
+        {
+          "dataElement": "MonthlyBirths",
+          "organisationUnit": "GR",
+          "period": "20190101",
```

Comment:
This doesn't seem correct, right? `MonthlyBirths` uses a `FINAL_EACH_MONTH` aggregation so one would expect this to be `201901`!

I believe this is a "bug" in this aggregation - as a matter of fact, most current aggregations almost never concern themselves with normalising the return period based on what they do. We'd need to change that to make them compatible with the indicator architecture - probably in https://github.com/beyondessential/tupaia-backlog/issues/2150

We were lucky in `PSSS`: we use week types there which force a period update (periods include a `W` in them so the conversion is inevitable for inclusion/comparison purposes etc)